lighthouse-core/test/test-utils.js

```diff
@@ -41,3 +42,38 @@ expect.extend({
     return toBeCloseTo.call(thisObj, ...args);
   },
 });
+
+function getProtoRoundTrip() {
+  let sampleResultsRoundtripStr;
+  let describeIfProtoExists;
+  let itIfProtoExists;
+  try {
+    sampleResultsRoundtripStr =
+      fs.readFileSync(__dirname + '/results/sample_v2_round_trip.json', 'utf-8');
+    describeIfProtoExists = describe;
+    itIfProtoExists = it;
+  } catch (err) {
+    if (process.env.GITHUB_ACTIONS) {
+      throw new Error('sample_v2_round_trip must be generated for CI proto test');
+    }
+    // Otherwise no proto roundtrip for tests, so skip them.
+    // This is fine for running the tests locally.
+
+    itIfProtoExists = (...args) => {
+      // eslint-disable-next-line no-console
+      console.warn('Skipping test - you need to run yarn test-proto first.');
```

Comment:
yeah:

<img width="729" alt="Screen Shot 2020-04-15 at 19 58 58" src="https://user-images.githubusercontent.com/316891/79400213-93fbfe80-7f53-11ea-8616-11c60dc21d61.png">

I feel like the `console.warn` is worse in the default local/travis case than having to look inside `proto-test` when you're trying it out and it's just skipping the test

Reply:
Without the warning, there's zero indication of where the skipped test was.

![image](https://user-images.githubusercontent.com/4071474/79505938-03243200-7fea-11ea-981d-0ea0ab1f12ab.png)

would it help to make the warning less strongly worded?

```
Skipped optional test - if you want to run proto tests, first install proto and run yarn test-proto
```

also could suppress this error in travis/appveyor ci

Reply:
the wording is fine (so is this new wording), it's more that it appears in the middle of the run of passed tests like a test failure and it takes mental parsing every time to realize it's fine.

I don't know, what do other people think of that?

(I also think it's fine that you'd see there was a basics action failure on `proto-test.js` and you'd open up the file and see the comment explaining the dependency on `yarn test-proto`. There is the issue of running locally and wondering why it skipped everything (and didn't fail), but `yarn unit-core` is run a whole lot more than `yarn jest proto-test`)